keyboards/kprepublic/bm65hsrgb_iso/bm65hsrgb_iso.c

```diff
@@ -56,4 +60,14 @@ void rgb_matrix_indicators_user(void) {
         rgb_matrix_set_color(30, 0xFF, 0xFF, 0xFF);
     }
 }
+
+void suspend_power_down_kb(void) {
+    rgb_matrix_set_suspend_state(true);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb(void) {
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
```

Comment:
```suggestion
```

These are now handled in the core and are probably not needed on keyboard level:
https://github.com/qmk/qmk_firmware/blob/master/quantum/quantum.c#L528